.swift-version

```diff
@@ -0,0 +1 @@
+3.0
```

Comment:
I could, I am just not sure that actually does anything. I have not found docs for `.swift-version` meaning, and even Nimble (and Alamofire) seem to specify `3.0` even though the latest version has been updated for Swift 3.1. So unless you are sure that specifying `3.1` there is the right thing to do, I would leave it at `3.0.`. I no longer am able to try things with Swift 3.0, so it would be probably hard to test with that release of the language.